netlify.toml

```diff
@@ -3,6 +3,6 @@
   publish = "dist"
 
 [[redirects]]
-  from = "/*"
+  from = "/"
```

Comment:
Not that it matters, but we should either leave the star for SPA behavior or just remove the rule all together (the vite SPA has no routes right now so we removing all together will work).

Reply:
Reverted this but added a `/prerender` path to call the API directly (useful for testing remote URLs)